bundles/binding/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/core/DmxTransmitter.java

```diff
@@ -37,11 +37,15 @@
 
     private boolean suspended;
 
+    private int refreshInterval = 0;
+    private long lastTransmit;
+
     /**
      * Default constructor.
      */
     public DmxTransmitter(DmxService service) {
         this.service = service;
+        lastTransmit = 0;
```

Comment:
Could `lastTransmit` be initialized to 0 above, instead of one private member being initialized that way, and the other initialized in the constructor?